Automattic/woocommerce-payments
# Redirect to WC home page after successful WCPay KYC

Fixes #1681

#### Changes proposed in this Pull Request

This changes onboard so that a successful KYC flow on Stripe's side results in redirecting to the WooCommerce home  page, rather than the WooCommerce Payments settings page.

- No tests were included because this doesn't really change the underlying business logic, just adding a different redirect. It would also be difficult to test because it relies on lots of services, and the implementation of redirects is difficult to test. 

#### Testing instructions

It's a bit involved, sorry.

1. Build a plugin zip with `npm run build:release`
2. create a new Jurassic Ninja site
3. SSH into the JN  site
4. `vim apps/user123/public/wp-config.php`
5. add `define( 'WCPAY_DEV_MODE', true )`
6. `esc : wq` (otherwise you could be stuck in vim until heat death of the universe)
7. Install WooCommerce, with a US based store so that WooCommerce Payments can be installed. However, deselect WCPay installation during onboarding (otherwise you'll have to uninstall it later
8. Install WooCommerce Payments using the zip file generated earlier
9. Set up WooCommerce Payments. Stripe should be in test mode.
10. Work through the KYC flow.
11. On completion you should be taken to the WooCommerce home page, instead of the WooCommerce Payments settings page.

Please read P7bbVw-3ww-p2 before opening the PR, assign the correct status to it, __and make sure that the related issue uses the Has PR status!__.
